### ISSUE TITLE
Readability scroll issue

### ISSUE THREAD

**Issue by kid1412621** [Ashinch/ReadYou#517]
**1. Environment**

-   Device: [Pixel 5]
-   OS: [Android 14]
-   APP: [0.9.10]

**2. Describe the bug**

After enabled readability, fastly scroll down to bottom, then slowly scroll up a little. Unable to scroll up anymore. 

https://github.com/Ashinch/ReadYou/assets/26278054/39e9b41a-441d-477d-b608-47caaa9186ab

**Comment by mbestavros** [comment]
@kid1412621 I'm unable to see the bug on my device. Are you seeing it when app language is set to English too?

**Comment by Ashinch** [comment]
This is a definite bug. The problem occurs in large pieces of composable items such as pictures. When scrolling to the screen again, it is re-rendered. Due to the characteristics of compose, we can't measure the height of all the composable items in the whole article at the same time, so I guess this problem cannot be avoided when using composable items to render the article. Similar to the causes of this problem are: scroll bar jitter.

（I hope I can explain it clearly)

I will provide users choice to render articles with WebView later.

At that time, such issues will be closed.

**Comment by kid1412621** [comment]
> @kid1412621 I'm unable to reproduce the bug on my device. Are you seeing it when app language is set to English too?

Yes, the app lang is English. Maybe lots of images in article will help to reproduce.


> This is a definite bug. The problem occurs in large pieces of composable items such as pictures. When scrolling to the screen again, it is re-rendered. Due to the characteristics of compose, we can't measure the height of all the composable items in the whole article at the same time, so I guess this problem cannot be avoided when using composable items to render the article. Similar to the causes of this problem are: scroll bar jitter.
> 
> （I hope I can explain it clearly)
> 
> I will provide users choice to render articles with WebView later.
> 
> At that time, such issues will be closed.

So this issue might happen even without Readability?

**Comment by Ashinch** [comment]
The exception has been caught, and the link to the article that caused the error can be found in the exception log. At that time, we can open another issue to discuss it.

**Comment by Ashinch** [comment]
> This is a definite bug. The problem occurs in large pieces of composable items such as pictures. When scrolling to the screen again, it is re-rendered. Due to the characteristics of compose, we can't measure the height of all the composable items in the whole article at the same time, so I guess this problem cannot be avoided when using composable items to render the article. Similar to the causes of this problem are: scroll bar jitter.
> 
> （I hope I can explain it clearly)
> 

#110

**Comment by Ashinch** [comment]
> So this issue might happen even without Readability?

Yes.

**Comment by Aaloone** [comment]
Is it possible to set it up to choose between using native components or WebView based on feed? Some sources have issues with text weight display when using WebView, while others have problems with photo display when using native components.
